ST_resgroups.c

```diff
@@ -717,17 +739,18 @@ void rgroup_IncrAges(void)
 /* Sums relsize for all individuals in all species in RGroup rg.
    param rg = RGroup index.
    Return: RGroup relsize. */
+
 RealF getRGroupRelsize(GrpIndex rg){
     Int n;
 	SppIndex sp;
 	double sum = 0.0;
 
     ForEachEstSpp( sp, rg, n){
-		sum += getSpeciesRelsize(sp);
+	sum += getSpeciesRelsize(sp);
     }
 
     if(RGroup[rg]->est_count > 0){
-        return (RealF) sum / (RealF) RGroup[rg]->est_count;
+        return (RealF) sum;
```

Comment:
If I remember correctly rgroup relsize has been divided by `est_count` since I've been on the project. Are we sure it should be changed?

Reply:
@chaukap thanks for asking. Yes, @dschlaep and I revisited this and we don't understand why it was ever divided by `est_count`. When simulating more than 1 species per rgroup with the old implementation, it will overestimate each individual's contribution to the rgroup size. See issue #366